packages/components/inset-text/template.njk

```diff
@@ -5,5 +5,9 @@
 {{- nhsukAttributes(params.attributes) }}>
   <span class="nhsuk-u-visually-hidden">Information: </span>
   {# params.HTML supported for backwards compatibility - see issue #950 #}
-  {{ (params.html or params.HTML) | safe }}
+  {% if params.html or params.HTML %}
+    {{ (params.html or params.HTML) | safe }}
+  {% else %}
+    <p>{{ params.text }}</p>
```

Comment:
This will end up outputting an empty paragraph if no args are passed - probably fine though?

Reply:
@edwardhorsford I think so? We don't really have any way of handling missing arguments gracefully across all the components.